test/unit/rubygem_searchable_test.rb

```diff
@@ -62,8 +62,8 @@ class RubygemSearchableTest < ActiveSupport::TestCase
       example_gem2 = create(:rubygem, name: "example_gem2", downloads: 1)
       example_gem3 = create(:rubygem, name: "example_gem3", downloads: 1)
       create(:version, rubygem: example_gem1)
-      create(:version, rubygem: example_gem2, description: 'some text and keyword')
-      create(:version, rubygem: example_gem3, summary: 'some keyword')
```

Comment:
It was a naive attempt to ensure that `avgFieldLength` and `fieldLength` don't affect the score and we test just the boost used on fields (name has the maxmium boost ^5, followed by summary ^3 ). Change was needed because default(in tests) for `summary` is `nil` and that for `description` is `Some awesome gem`.
I have changed it to use same length for all the three fields. [Here](https://gist.github.com/sonalkr132/78bd0ce8c2dcdca5f1e0ac3e21f7efc0) is explain of the current implementation without boost.